snuba/reader.py

```diff
@@ -96,3 +98,30 @@ def execute(
             ),
             with_totals=with_totals,
         )
+
+
+class HTTPReader(Reader):
+    def __init__(
+        self, host: str, port: int, options: Optional[Mapping[str, str]] = None
+    ):
+        self.__base_url = f"http://{host}:{port}/"
+        self.__options = options if options is not None else {}
+
+    def execute(
+        self,
+        query: str,
+        settings: Optional[Mapping[str, str]] = None,
+        query_id: Optional[str] = None,
+        with_totals: bool = False,
```

Comment:
`with_totals` doesn't have any effect here, since the response comes back in the format we want already.